cypress/e2e/management-users.feature

```diff
@@ -0,0 +1,35 @@
+Feature: List management role changing
```

Comment:
I think we need another test here to ensure a user that doesn't have the SuperAdmin role is redirected to an error page if they access the user list or attempt to edit a user by hacking the URL params.  See here

**User list:** 
https://github.com/UKForeignOffice/lists/blob/a51bdb9d9fc7545d107027b811a8a59e7b8385db/src/server/components/dashboard/router.ts#L35

**User edit:** 
https://github.com/UKForeignOffice/lists/blob/a51bdb9d9fc7545d107027b811a8a59e7b8385db/src/server/components/dashboard/router.ts#L41

We can do the same thing for editing lists but that can be covered by the separate ticket you created.